lib/suse/connect/connection.rb

```diff
@@ -40,9 +40,10 @@ def initialize(endpoint, language: nil, insecure: false, debug: false, verify_ca
       end
 
       VERB_TO_CLASS.each_key do |name_for_method|
-        define_method name_for_method do |path, auth: nil, params: {}|
+        define_method name_for_method do |path, auth: nil, params: {}, version: SUSE::Connect::Api::VERSION|
```

Comment:
Following our discussion yesterday, I don't think it's worth it to add all this logic about versioning...
The package search API will respond fine even if you provide "v4", and when we decide to make the API "official" it will be aligned with the connect API version as well.

Reply:
Yes right, tought I already did the work and its not harming anybody. But yes I will remove it

Reply:
Well, it's more code that can have bugs, that has to be understood, that has to be tested.
I think if we don't use it, it's better not to have it.